code/drasil-docLang/lib/Drasil/Sections/TableOfAbbAndAcronyms.hs

```diff
@@ -18,6 +18,7 @@ select (x:xs) = case getA x of
 
 -- | The actual table creation function.
 tableAbbAccGen :: (Idea s) => [s] -> LabelledContent
+tableAbbAccGen [] = llcc tableAbbAccRef $ Paragraph (S "There are no" +:+ plural abbAcc)
```

Comment:
Can this be represented using the pattern I mentioned before or is this completely different? As is, there is no trailing period like in the other examples; using a common helper function would reduce the number of potential errors like this:
![image](https://github.com/JacquesCarette/Drasil/assets/35857611/6a0e5f2b-a6b4-4cb5-b32b-b64d87d2747b)

Reply:
Could you please clarify which pattern are you referring to @samm82?

Reply:
Oh ok, I don't think we can for this one because the pattern will convert the sentence to contents but the `Paragraph` section needs to have a `Sentence` as its input. However, I think to fix this, I just needed to change `+:+` to `+:+.`.